synapse/cortex.py

```diff
@@ -4477,11 +4491,24 @@ async def _ctorLayr(self, layrinfo):
         return await s_layer.Layer.anit(self, layrinfo)
 
     async def _initCoreLayers(self):
+        remlayrs = set()
         node = await self.hive.open(('cortex', 'layers'))
-        for _, node in node:
+        for layriden, node in node:
             layrinfo = await node.dict()
+
+            layrdirn = s_common.gendir(self.dirn, 'layers', layriden)
+            path = s_common.genpath(self.dirn, 'layer_v2.lmdb')
```

Comment:
This seems incorrect? Am i missing something?

Reply:
should be layrdirn, fixed